File: 136-single-number/136-single-number.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        c = {}
        for num in nums:
            if num in c.keys():
                c[num] = 2
            else:
                c[num] = 1
        
        for key, value in c.items():
            if value == 1:
                return key

File: 136-single-number/136-single-number.swift
class Solution {
    func singleNumber(_ nums: [Int]) -> Int {
        var singleNum = 0
        
        var dictionary = [Int: Int]()
        
        for num in nums {
            dictionary[num] = (dictionary[num] ?? 0) + 1
        }
        
        for item in dictionary {
            if item.value == 1 {
                singleNum = item.key
            }
        }
        return singleNum
    }
}